libs/fairness/src/lib/Controls/DropdownBar.styles.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  getTheme,
+  IProcessedStyleSet,
+  IStyle,
+  mergeStyleSets,
+  FontWeights,
+  FontSizes
+} from "office-ui-fabric-react";
+
+export interface IDropdownBarStyles {
+  frame: IStyle;
+  spinner: IStyle;
+  doneButton: IStyle;
+  infoButton: IStyle;
+  modalContentHelp: IStyle;
+  modalContentHelpText: IStyle;
+  toolTipWrapper: IStyle;
+  mainRight: IStyle;
+  actionButton: IStyle;
+  title: IStyle;
+  callout: IStyle;
+}
+
+export const DropdownBarStyles: () => IProcessedStyleSet<
+  IDropdownBarStyles
+> = () => {
+  const theme = getTheme();
+  return mergeStyleSets<IDropdownBarStyles>({
+    actionButton: {
+      height: "auto"
+    },
+    callout: {
+      padding: "20px 24px",
+      width: 320
+    },
+    doneButton: {
+      color: theme.semanticColors.bodyText,
+      fontSize: FontSizes.large,
+      fontWeight: FontWeights.regular,
+      height: "44px",
+      lineHeight: "24px",
+      margin: "auto",
+      padding: "12px"
+    },
+    frame: {
+      display: "flex",
+      flex: 1,
+      flexDirection: "column"
+    },
+    infoButton: {
+      border: "1px solid",
+      borderRadius: "50%",
+      color: theme.semanticColors.bodyText,
+      float: "left",
+      fontSize: "12px",
```

Comment:
From PR meeting:
Use things like `<Text block variant="xLarge"` in the actual code